New package: SteamWebAPIs v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: SteamWebAPIs
- Repository: https://github.com/NeroBlackstone/SteamWebAPIs.jl
- Created by: @NeroBlackstone
- Version: v0.1.0
- Commit: 5af12126551137359b8a70b23271fcfca0a8cdc9
- Reviewed by: @NeroBlackstone
- Reference: https://github.com/NeroBlackstone/SteamWebAPIs.jl/issues/1#issuecomment-2081105746
- Description: (WIP) Steam Web API wrapper for Julia
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835987043b3efa4dd873e2188de256353d718a3c9d558c82d335702ed67dac5ec2e0d19945fc52ed6aa47361fad865e6d9c3b4ce6718da5bb26b18a5365a82cfc09b207733c5dec8d5d5eb8939225d8ad173cfac39c0b4b7b03d3d3d3e431edfb252ca0e9a4ab1cef03bbbe7cf32d2e028e860d801f5a2c01970a737e03873061d29e -->

goerz commented:
[noblock]

LGTM, but of course, you have to fix the issues flagged by the bot. When you do, just repeat the registration comment, and the bot will automatically update this PR. Do not change the version number: that would create a new registration.

NeroBlackstone commented:
Please give me some time to make some changes and retrigger the Registrator. Thanks for your patient, it's my first time publish package.

goerz commented:
[noblock] There's no rush! It's entirely up to you. The bot will merge automatically after the issues have been fixed and the three-day waiting period has elapsed. You'll also have to edit your comment to include `[noblock]`, because otherwise, that'll prevent the bot from merging

NeroBlackstone commented:
[noblock]
@JuliaRegistrator

NeroBlackstone commented:
[noblock]
@JuliaRegistrator register()

JuliaRegistrator committed:
New package: SteamWebAPIs v0.1.0

UUID: e6ff7510-e9ee-4fc9-8d54-48d13dde7e75
Repo: https://github.com/NeroBlackstone/SteamWebAPIs.jl.git
Tree: 77a43138a36f965c470332d8f5483b1701f920d0

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
